CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 cmake_minimum_required(VERSION 3.6)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Should be set in toolchain:
* https://docs.hunter.sh/en/latest/overview/customization/toolchain-id.html#simple-toolchains